Копорушкин Данил

darl0ck opened the pull request:
init
[Посмотреть решение](https://rawgit.com/darl0ck/markup-task-4/master/index.html)

darl0ck committed:
Копорушкин Данил

init

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

darl0ck committed:
Выгнал прыгскок команду

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bazhenova commented:
Привет. Проявил фантазию, молодец) И все же будут замечания по оформлению:
1. Хочу тыкать не только на переключатель для списка, но и на текст рядом с ним

2. Цены почти не видно. Сделай заметными. И нигде не увидела старой цены. А как же скидки?!

3. Эффект при наведении на котика прикольный. А вот при наведении на имя, категорию всё начинает скакать. И еще не реализован эффект на карточку в целом

4. Где же длинные имена котиков? Хочу увидеть красивое обрезание

5. Раз ты сделал дополнительное задание, то возникает вопрос. Что будет, если высота картинок с котиками будет разной? Будут ли они выровнены по нижней линии? 

6. >обратите внимание, что есть оценка в ползвезды

У тебя не смогла такого проставить

bazhenova commented:
🍅

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

darl0ck committed:
Исправил ошибки

- Исправил
- Навел красоту
- И еще разобрался в Gulp))) (чтобы префиксы раставлять азазаз)

darl0ck commented:
@bazhenova 
🔔 
Аня спаси 
Хрюнжель пишет , что не ок , а трэвису все ок 👎

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

nikshel commented:
@bazhenova @darl0ck 
Теперь все океюшки

bazhenova commented on index.html:
```diff
@@ -3,7 +3,224 @@
 <body>
+<input id="changeArrangement" type="checkbox">
+<label for="changeArrangement">На столбик</label>
+<div class="to-be-changed" >
```
Название класса не очень

bazhenova commented on index.html:
```diff
@@ -3,7 +3,224 @@
 </head>
 <body>
+<input id="changeArrangement" type="checkbox">
+<label for="changeArrangement">На столбик</label>
```
При именовании - никакого camelCase. Либо просто слово, либо слова через `-`

bazhenova commented on index.html:
```diff
@@ -3,7 +3,224 @@
+<label for="changeArrangement">На столбик</label>
+<div class="to-be-changed" >
+    <!-- Здесь прикладывать основное усилие -->
+    <div class="block">
```
А в чем смысл такого названия класса? Блок чего?

bazhenova commented on index.html:
```diff
@@ -3,7 +3,224 @@
+        <p class="newPrice">$30</p>
+    </div>
+
+    <div class="block">
```
Добавь разметке больше семантики

bazhenova commented on index.html:
```diff
@@ -3,7 +3,224 @@
+    <div class="block">
+        <a href="#" class="cat2 change"></a>
+        <div class="element">
+        <p class="name">Любоффка-кот</p>
```
Снова едут отступы. Проверь весь файл. Много где такое

bazhenova commented on index.html:
```diff
@@ -3,7 +3,224 @@
+        <p class="oldPrice">$1000</p>
+        <p class="newPrice">$999</p>
+    </div>
+
```
Пустые строки

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+
+.cat6
+{
+    background: url(satisfiedcat.jpg);
```
Все картинки сделать через  тег `img`. И не забудь указать `alt` и `title`

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+
+.block
+{
+    margin: 61px 61px 13px;
```
Без этого выглядит красивше

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+    height: 325px;
+    padding: 50px;
+    text-align: left;
+    /*width: 190px;*/
```
😑

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+.cat2
+{
+    background: url(lovelycat.jpg);
+    display: block;
```
Три строчки одинаковы для всех котов, избавься от дублирования

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+.price
+{
+    font-family: 'Pacifico', cursive;
+    /*position: absolute;*/
```
😑

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+.mark
+{
+    font-size: larger;
+
```
Пустая строка

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+.rating > input:checked + label:hover,
+.rating > input:checked + label:hover ~ label,
+.rating > input:checked ~ label:hover,
+.rating > input:checked ~ label:hover ~ label,
```
Очень похожи 390-391 и 392-393 строчки. Расскажи, в чем разница между `+` и `~`

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+
+.rating:not(:checked) > label:before
+{
+    content: '❤ ';
```
Раз ты решил использовать сердечки, то ненужные картинки со звездочками надо удалить

darl0ck replied:
@bazhenova 
прошу помочь если это легально с рейтингом в пол-звезды(сердца)
Для этого мне нужно полностью все переделывать?
:(((

bazhenova commented on index.css:
```diff
@@ -0,0 +1,404 @@
+
+input[type='checkbox']:not(:checked) ~ .to-be-changed .block
+{
+    height: 325px;
```
Зачем переопределять тем же самым значением? (См. 77 строку)

bazhenova reviewed (commented):
1. Цены все еще видно плохо
![image](https://user-images.githubusercontent.com/12410663/32405854-02887ef8-c18f-11e7-9877-a47ffe9a3553.png)

2. Если ты все-таки делаешь доп. задание, то сделай хотя бы одну картинку другой высоты, чтобы было заметно выравнивание по нижней линии

3. Не могу проставить оценку в полсердца
>обратите внимание, что есть оценка в ползвезды

4. При выводе списком у котиков слишком много пустого места
![image](https://user-images.githubusercontent.com/12410663/32405871-8eaf7076-c18f-11e7-9d5d-1447a7521ebc.png)

🍅

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bazhenova commented on index.css:
```diff
@@ -0,0 +1,344 @@
+}
+
+input[type='checkbox']:checked ~ .main-div .wrapper .price,
+input[type='checkbox']:checked ~ .main-div .wrapper .oldPrice
```
Нужно ли указывать столько классов?

bazhenova commented on index.css:
```diff
@@ -0,0 +1,344 @@
+.about:hover
+{
+    color: #b36370;
+
```
Пустые строки в некоторых местах файла

darl0ck replied:
У меня все норм когда заливаю все отступает 😔😔😔😔

bazhenova commented on index.css:
```diff
@@ -0,0 +1,344 @@
+@media only screen and (max-device-width: 768px)
+{
+    .title
+{
```
Поехал отступ. Проверь по всему файлу

bazhenova commented on index.html:
```diff
@@ -3,7 +3,234 @@
 <body>
+<input id="change-arrangement" type="checkbox">
+<label for="change-arrangement">На столбик</label>
+<div class="main-div">
```
Это выглядит как не `div`, а `main`

bazhenova reviewed (commented):
Котики стали очень широкими
![image](https://user-images.githubusercontent.com/12410663/32416985-fa6d28dc-c273-11e7-8c10-d42f5b0b1257.png)
🍅 

P.S. написала в слак

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bazhenova commented on index.css:
```diff
@@ -0,0 +1,371 @@
+/*stylelint-disable max-line-length*/
```
😑

bazhenova commented on index.css:
```diff
@@ -0,0 +1,371 @@
+
+input[type='checkbox']:checked ~ main a img
+{
+    height: auto !important;
```
Можно ли обойтись без `!important`?

darl0ck replied:
Можно и без него 
Но я сделал с ним что бы при переключении картинки не выглядела как сжатая

bazhenova reviewed (commented):
Когда навожу на фото котика, то он исчезает(
![image](https://user-images.githubusercontent.com/12410663/32440243-2910b8f2-c314-11e7-851e-b8f2066c364e.png)

Будешь доделывать половинки сердец?

🍅

darl0ck commented:
@bazhenova Привет 
Да сейчас буду доделывать
Очень странно, что кот пропадает
У меня все нормально грузится , мб просто сервер тупитт и долго картинку выводит

darl0ck committed:
Вроде всё

Каеф

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

darl0ck committed:
Вот там сплоховал гит((((((

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

darl0ck commented:
@bazhenova 
Не понимаю почему тут старый отображается в "Посмотреть Решение"
Надеюсь что все ок

bazhenova commented:
>Задание необходимо решить без использования JS

Вижу закоммиченный `js` файл, атата. Молодец, что разобрался. А без него никак?)

darl0ck commented:
@bazhenova 
Ахаха
Да это ж гульп 
Просто было лень ручками префиксы ставить я и настроил его
ахах
Случайно его приложил((

bazhenova commented:
Сейчас прощаю) Обленились студенты совсем.. Принято!
